File: app/src/main/java/com/mdgd/pokemon/ui/pokemons/infra/CharacteristicComparator.java
package com.mdgd.pokemon.ui.pokemons.infra;

import com.mdgd.pokemon.models.repo.dao.schemas.PokemonFullDataSchema;

public interface CharacteristicComparator {
    int compare(PokemonFullDataSchema p1, PokemonFullDataSchema p2);
}

File: app/src/main/java/com/mdgd/pokemon/ui/pokemons/infra/CharacteristicComparator.kt
package com.mdgd.pokemon.ui.pokemons.infra

import com.mdgd.pokemon.models.repo.dao.schemas.PokemonFullDataSchema

interface CharacteristicComparator {
    fun compare(p1: PokemonFullDataSchema, p2: PokemonFullDataSchema): Int
}
